pyqt-apps/siriushla/si_ap_orbintlk/custom_widgets.py

```diff
@@ -362,7 +362,7 @@ def _setupUi(self):
                 text = '\n\nBPM calculation consider the angulation\n' \
                     'estimative:' \
                     '\n\n(pos @downstream - pos @upstream)\n\n' \
```

Comment:
I think it would look better without this space between `pos` and `@`

Reply:
done!